.github/workflows/release.yml

```
@@ -26,12 +23,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
       - run: echo '${{ secrets.ENCODED_RELEASE_KEYSTORE }}' | base64 --decode > android/playstore.keystore
+      - name: Add Flutter to PATH
+        run: echo "/home/ubuntu/flutter/bin" >> $GITHUB_PATH
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:36: Double quote to prevent globbing and word splitting

(shellcheck)